changelogs/unreleased/7531-race-condition-in-exporter-file-upload.yml

```diff
@@ -0,0 +1,10 @@
+description: >
+  The upload_file endpoint will now silently ignore attempts to upload a file if a file with the same
+  hash was previously uploaded.
+issue-nr: 7531
+change-type: minor
+destination-branches: [master, iso7]
```

Comment:
Is there a specific reason why we don't want this fix into iso6?

Reply:
Hmm no actually. It felt more like a minor than a patch to me, but looking back again at it, this shouldn't drive to which branch it gets merged. I'll add iso6 and I'll try to be more careful about this in the future.